RoddyCore/src/de/dkfz/roddy/core/ProjectLoader.groovy

```diff
@@ -457,8 +459,36 @@ class ProjectLoader {
                 sb << "  " << projectID << "@" << aID << Constants.ENV_LINESEPARATOR;
             }
             throw new ProjectLoaderException(sb.toString());
-        } else if (analysis.getRuntimeService() == null) {
+        }
+
+        if (analysis.getRuntimeService() == null) {
             throw new ProjectLoaderException("There is no runtime service class set for the selected analysis. This has to be set in either the project configuration or the analysis configuration.");
         }
+
+        List<String> errors = []
+
+        // Earliest check for valid input and output directories. If they are not accessible or writeable.
+        // Start with the input directory
+        for (File _dir = analysis.getInputBaseDirectory(); _dir; _dir = _dir.parentFile) {
+            if (!FileSystemAccessProvider.instance.isReadable(_dir) || !FileSystemAccessProvider.instance.isExecutable(_dir)) {
+                errors << "The input directory was not readable at path ${_dir}."
+                break
+            }
+        }
+
+        for (File _dir = analysis.getOutputBaseDirectory(); _dir; _dir = _dir.parentFile) {
+            if (!FileSystemAccessProvider.instance.isReadable(_dir) || !FileSystemAccessProvider.instance.isExecutable(_dir))
+                if (!FileSystemAccessProvider.instance.isReadable(_dir)) {
```

Comment:
You only report, if it is not readable? You test for executability but don't report, if it not executable?